app/models/restaurant.rb

```diff
@@ -0,0 +1,6 @@
+class Restaurant < ApplicationRecord
+  validates :name, presence: true
+  validates :latitude, presence: true
+  validates :longitude, presence: true
+  validates :id, presence: true
```

Comment:
We won't need line 5 unless we are supposed to be supplying an `id` before a restaurant is persisted to the database